src/test/java/seedu/realtime/testutil/TypicalClients.java

```diff
@@ -60,10 +60,10 @@ public class TypicalClients {
     private TypicalClients() {} // prevents instantiation
 
     /**
-     * Returns an {@code AddressBook} with all the typical Clients.
+     * Returns an {@code realTime} with all the typical Clients.
      */
-    public static AddressBook getTypicalAddressBook() {
-        AddressBook ab = new AddressBook();
+    public static RealTime getTypicalRealTime() {
+        RealTime ab = new RealTime();
```

Comment:
```suggestion
    public static RealTime getTypicalRealTime() {
        RealTime rt = new RealTime();
```
Doing this means the next few lines of code has to change as well